composeApp/src/commonMain/kotlin/sp/bvantur/tasky/core/ui/theme/TaskyTypography.kt

```diff
@@ -0,0 +1,53 @@
+package sp.bvantur.tasky.core.ui.theme
+
+import androidx.compose.material3.Typography
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import org.jetbrains.compose.resources.Font
+import tasky.composeapp.generated.resources.Res
+import tasky.composeapp.generated.resources.inter_bold
+import tasky.composeapp.generated.resources.inter_extralight
+import tasky.composeapp.generated.resources.inter_light
+import tasky.composeapp.generated.resources.inter_regular
+import tasky.composeapp.generated.resources.inter_thin
+
+@Composable
+private fun interFamily() = FontFamily(
+    Font(Res.font.inter_light, FontWeight.Light, FontStyle.Normal),
+    Font(Res.font.inter_regular, FontWeight.Normal, FontStyle.Normal),
+    Font(Res.font.inter_thin, FontWeight.Normal, FontStyle.Italic),
+    Font(Res.font.inter_extralight, FontWeight.Medium, FontStyle.Normal),
+    Font(Res.font.inter_bold, FontWeight.Bold, FontStyle.Normal)
+)
```

Comment:
If it's just about getting a value without any parameters, could it make sense to just make this a simple variable?

Reply:
I would prefer having it as a variable, but we are using a **Font()** component which is a **Composable** component. And there is an error for it like this:

`@Composable invocations can only happen from the context of a @Composable function`

If there is some other way to overcome this restriction let me know. I didn't find a solution to use it as a variable.